include/counter.h

```diff
@@ -132,19 +254,125 @@ static inline int counter_set_alarm(struct device *dev,
  *
  * @param dev Pointer to the device structure for the driver instance.
  *
- * @retval 1 if the counter interrupt is pending.
+ * @retval 1 if the counter any interrupt is pending.
```

Comment:
The change made this description incorrect.

Reply:
fixed